compiler/src/lnntoamm/Expr.ts

```diff
@@ -385,15 +389,18 @@ class Call extends Expr {
     fns: Fn[],
     maybeClosure: VarDef | null,
     args: Ref[],
+    scope: Scope,
   ) {
+    // console.log('~~~ generating call ', ast, fns, maybeClosure, args, scope);
```

Comment:
Left behind debug log. Do you need this often or should it be deleted.

Reply:
i end up using it quite often